src/screens/ExchangeModal.js

```diff
@@ -292,9 +294,10 @@ export default function ExchangeModal({
   } = useSwapDerivedOutputs(chainId, type);
 
   const lastTradeDetails = usePrevious(tradeDetails);
+  const isSufficientBalance = useSwapIsSufficientBalance(inputAmount);
 
   const {
-    isHighPriceImpact,
+    isHighPriceImpact: useIsHighPriceImpact,
```

Comment:
dont know how I feel about the naming here makes it seem like a hook

Reply:
rightt